src/main/java/com/ss/lms/entity/CopiesId.java

```diff
@@ -0,0 +1,74 @@
+package com.ss.lms.entity;
+
+import java.io.Serializable;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+import org.springframework.stereotype.Component;
+@Component
+@Embeddable
```

Comment:
ohhh i see what you did now. Why did you choose to go this route instead of mapping it in the copies itself. Is it beacuse its a three key value relationship???

Reply:
Yes. This is needed for compound key.